vllm/entrypoints/openai/api_server.py

```diff
@@ -72,6 +77,60 @@ async def _force_log():
     yield
 
 
+@asynccontextmanager
+async def build_backend(args) -> AsyncIterator[VLLMBackend]:
+    # Context manager to handle backend lifecycle
+    # Ensures everything is shutdown and cleaned up on error/exit
+    global engine_args
+    engine_args = AsyncEngineArgs.from_cli_args(args)
+
+    # Backend itself still global for the silly lil' health handler
+    global backend
+
+    # First need to determine if this is an embeddings model
+    # (no remote backend for those)
```

Comment:
why?

Reply:
cc @robertgshaw2-neuralmagic 

I think because it'd be more work to hook those up, and we're not worried about the throughput improvements for embeddings atm

Reply:
Planning to do follow to enable RPC for Embedding

Reply:
leave a comment saying future work plz.